erpnext/accounts/doctype/chart_of_accounts_importer/chart_of_accounts_importer.py

```diff
@@ -158,36 +163,47 @@ def return_parent(data, child):
 
 				return [child] + parent_account_list
 
-	charts_map, paths = {}, []
+	charts_map = defaultdict(dict)
+	paths = []
 
 	line_no = 3
 	error_messages = []
 
-	for i in data:
-		account_name, dummy, account_number, is_group, account_type, root_type = i
+	for row in data:
+		account_name, parent_account, account_number, is_group, account_type, root_type, *others = row
```

Comment:
This is failing on python2.7 and erroring it out

Reply:
Seems like that syntax is not supported on 2.7. I can change it, but are we supporting that version though?

Reply:
Ideally we should be dropping it, but I will also have to change Travis config to not run 2.7 tests.